cmake/developer_package/api_validator/api_validator.cmake

```diff
@@ -71,24 +71,11 @@ function(_ov_add_api_validator_post_build_step)
     # collect targets
 
     _ie_add_api_validator_post_build_step_recursive(TARGET ${API_VALIDATOR_TARGET})
-
-    # remove targets which were tested before
-    foreach(target IN LISTS API_VALIDATOR_TARGETS)
```

Comment:
@ilya-lavrenov the logic is removed because in case if target is presented several times, after filtering it will be pinned to only one target (e.g. if apply `ie_add_api_validator_post_build_step()` to samples, TBB will be presented only with `sync_bechmark`, but should be presented for each sample). It may be useful for CI, but incorrect for local runs